doc/source/prometheus.rst

```diff
@@ -0,0 +1,31 @@
+====================
+ Prometheus support
+====================
+
+`Prometheus`_ can use Gnocchi to store its data through
+`Remote Write Adapter`_.
+
+Example of Prometheus configuration::
+
+  remote_write:
+  - url: "http://localhost:8041/v1/prometheus/write"
+
+
+The `/v1/prometheus/write` handles the `WriteRequest` protobuf message
+format of Prometheus.
+
+Each metrics received in the message, we take the name, two attributes (`job`
+and `instance`) and the timeseries.
+
+Gnocchi uses these attributes to identify the orignal resource polled by
+Prometheus.
+
+For each job/instance pair, we have a corresponding resource in Gnocchi.
+Prometheus resources are stored within the `prometheus` resource-type.
+The `original_resource_id` field is built like `<job>@<instance>`.
+A `prometheus` resource-type have to extra attributes `job` and `instance`.
+
+Then the `name` of the `Prometheus`_ metric is used as Gnocchi metric name.
+
```

Comment:
Ok I think I get what you mean in this page. Here's a clearer version:


Gnocchi maps Prometheus metrics to its data model.

For each metric sent by Prometheus, Gnocchi maintains a corresponding resource based on each the `job` and `instance` pair. This resource is created with the `prometheus` resource type and contains two attributes, `job` and `instance`. The metrics sent by Prometheus with this pair are attached to that resource and filled with the provided measures.

Reply:
done